File: wesignmanagement-client-DEV/src/app/models/requests/frequency-report-request.ts
import { ReportFrequency } from "src/app/enums/report-frequency";
import { ReportParameters } from "../report-parameters";
import { ReportMode } from "src/app/enums/ReportMode.enum";

export class FrequencyReportRequest {
    public reportParameters: ReportParameters;
    public frequency: ReportFrequency;
    public reportType: ReportMode;
    public emailsToSend: string[];
}

File: user-backend/WeSignManagement/Request/FrequencyReportRequest.cs
using Common.Enums.Reports;
using Common.Models.ManagementApp.Reports;
using System.Collections.Generic;

namespace WeSignManagement.Request
{
    public class FrequencyReportRequest
    {
        public ReportParameters ReportParameters { get; set; }
        public ManagementReportFrequency Frequency { get; set; }
        public ManagementReportType ReportType { get; set; }
        public List<string> EmailsToSend { get; set; }
    }
}
